parkings/api/operator/parking.py

```diff
@@ -95,3 +115,79 @@ def perform_create(self, serializer):
 
     def get_queryset(self):
         return super().get_queryset().filter(operator=self.request.user.operator)
+
+
+class OperatorAPIParkingLotInfo(GenericAPIView):
+    """
+    Fetch information with a POST request about a specific parking lot with either location coordinates
+    or terminal number. The information includes the location (coordinates), zone, terminal
+    number and a list of parking rules related to the queried parking lot.
+    """
+    permission_classes = [OperatorAPIParkingPermission]
+    serializer_class = ParkingLotInfoSerializer
+    http_method_names = ['post']
+
+    def post(self, request):
+        location = request.data.get('location')
+        terminal_number = request.data.get('terminal_number')
+
+        if not location and not terminal_number:
+            raise ParseError(detail='No search parameters entered.')
+
+        terminal_obj = self._get_terminal_object(
+            coordinates=location.get('coordinates') if location else None,
+            terminal_number=terminal_number if terminal_number else None)
+        data = self._get_serializer_data(terminal_obj)
```

Comment:
I think we should still try to query the data even if there is no terminal, i.e. with bare location info. You could still e.g. tell the zone number for the coordinates.